FirebaseFunctionsSwift.podspec

```diff
@@ -35,20 +35,16 @@ Swift SDK Extensions for Cloud Functions for Firebase.
     'FirebaseFunctionsSwift/Sources/**/*.swift',
   ]
 
-  s.dependency 'FirebaseFunctions', '~> 8.11'
-  s.dependency 'FirebaseSharedSwift', '~> 8.11'
-
-  # Using environment variable because of the dependency on the unpublished
-  # FirebaseFunctionsTestingSupport.
```

Comment:
Side note: we should see if we can now get rid of this in other products. No action needed for this PR.

Reply:
Agreed. We can also get rid of all the Objective C soon :)